core/domain/customization_args_util_test.py

```diff
@@ -376,9 +376,9 @@ def test_frontend_customization_args_defs_coverage(self):
         for line in lines:
             # Search for XCustomizationArgsBackendDict interfaces and extract X,
             # where X is a interaction id.
-            # Group 1: The characters 'interface'.
-            # Group 2: The interaction id.
-            # Group 3: The characters 'CustomizationArgsBackendDict'.
+            # Match 1: The characters 'interface'.
+            # Match 2: The interaction id.
+            # Match 3: The characters 'CustomizationArgsBackendDict'.
```

Comment:
Group is the correct terminology here actually, it's characters that needs to change:

```suggestion
            # Group 1: The match for 'interface'.
            # Group 2: The match for interaction id.
            # Group 3: The match for 'CustomizationArgsBackendDict'.
```

Reply:
Gotchu, done